### ISSUE TITLE
[BUG] (mostly harmless) error when cancelling .csv and .mat file opening in PYMEVis

### ISSUE THREAD

**Issue by csoeller** [python-microscopy/python-microscopy#1433]
**Describe the bug**
An error is raised when the tabular dialogs for `.csv` and `.mat` files are canceled

Note: error is largely inconsequential but should ideally be avoided. Fix is quite simple, PR to follow...

**To Reproduce**
1. Ty opening `.cvs` file in PYMEVis
2. Cancel tabular dialogs that show field mapping 

**Error Message**

```
Traceback (most recent call last):

  File "/Users/csoe002/miniconda3/envs/pyme38_2/lib/python3.8/site-packages/wx/core.py", line 2346, in Notify

    self.notify()

  File "/Users/csoe002/miniconda3/envs/pyme38_2/lib/python3.8/site-packages/wx/core.py", line 3552, in Notify

    self.result = self.callable(*self.args, **self.kwargs)

  File "/Users/csoe002/Documents/src/PYMEsrc/pyme38/PYME/LMVis/visCore.py", line 800, in OpenFile

    self.pipeline.OpenFile(filename, **args)

TypeError: OpenFile() argument after ** must be a mapping, not NoneType
```

**Expected behavior**
No error.

**Comment by David-Baddeley** [comment]
@csoeller - can we close this now #1434 has been merged?

**Comment by csoeller** [comment]
fixed with #1434
